manager/controlapi/network.go

```diff
@@ -18,34 +18,45 @@ func validateIPAMConfiguration(ipamConf *api.IPAMConfig) error {
 		return grpc.Errorf(codes.InvalidArgument, "ipam configuration: cannot be empty")
 	}
 
+	var me MultiError
+
 	_, subnet, err := net.ParseCIDR(ipamConf.Subnet)
 	if err != nil {
-		return grpc.Errorf(codes.InvalidArgument, "ipam configuration: invalid subnet %s", ipamConf.Subnet)
+		err = grpc.Errorf(codes.InvalidArgument, "ipam configuration: invalid subnet %s", ipamConf.Subnet)
```

Comment:
Thanks @aaronlehmann. Actually not all err's code is `codes.InvalidArgument`, there is also `codes.Unimplemented`.